lib/api/lambda.js

```diff
@@ -100,6 +100,17 @@ Lambda.prototype.scheduledEvent = function(path, handler) {
 };
 
 /**
+ * Routes API GATEWAY AUTHORIZER EVENT requests for a given rule to the handler. 
+ * Rule names will be the method and path, e.g. '/GET/auth'
```

Comment:
Can you provide a link to the docs here? maybe to http://docs.aws.amazon.com/apigateway/latest/developerguide/use-custom-authorizer.html (or if there's a better page, that one)